test/bats/akeyless.bats

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bats
+
+ load helpers
+
+ BATS_TESTS_DIR=test/bats/tests/akeyless
+ WAIT_TIME=120
+ SLEEP_TIME=1
+
+
+ export LABEL_VALUE=${LABEL_VALUE:-"test"}
+ export ANNOTATION_VALUE=${ANNOTATION_VALUE:-"app=test"}
```

Comment:
These values are not used anywhere in this test suite. Could you remove them?

Reply:
fixed